src/devices/cpu/z80/z80.cpp

```diff
@@ -3473,6 +3492,7 @@ void z80_device::device_start()
 	save_item(NAME(m_busrq_state));
 	save_item(NAME(m_after_ei));
 	save_item(NAME(m_after_ldair));
+	save_item(NAME(m_m_shared));
```

Comment:
Is this supposed to have two `m_` prefixes on it?

Reply:
Yes m-cycle of instruction - var shared between M-states

Reply:
Ah okay.